app/controllers/api/v1/subject_sets_controller.rb

```diff
@@ -86,7 +86,6 @@ def destroy_relation(resource, relation, value)
   private
 
   def remove_linked_set_member_subjects(resource, set_member_subjects)
-    SubjectWorkflowCount.where(set_member_subject: set_member_subjects).delete_all
```

Comment:
no performance consideration. principle of least surprise and the functionality being visible in code.